gb-backend/grails-app/controllers/nl/thehyve/gb/backend/QueryController.groovy

```diff
@@ -6,6 +6,7 @@
 
 package nl.thehyve.gb.backend
 
+import javassist.NotFoundException
```

Comment:
I'd not suggest depending on the 3d party library for that.

Reply:
ah thanks for noticing! Will replace it